pkgs/development/python-modules/truststore/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, python3
```

Comment:
For pythonPackages `python3.pkgs` contents should be taken directly as input to the package function, instead of having `python3` as an input and accessing from there.

So: `buildPythonPackage, flit-core, aiohttp, httpx, pyopenssl, requests, trustme` should be directly listed here, `python3` removed, and the `python3.pkgs` references removed as well. See other examples in `development/python-modules/*`.

Reply:
Because using `python3` here will pin it to just `python310` right now, and make it not work on `python311`.

Reply:
Fixed, thanks for the review